x-pack/plugins/cases/common/ui/types.ts

```diff
@@ -12,28 +12,28 @@ import type {
   READ_CASES_CAPABILITY,
```

Comment:
suggestion: Not in this PR but shall we rearrange this folder from `x-pack/plugins/cases/common/ui/types.ts` to `x-pack/plugins/cases/common/types/ui/index.ts` ? 
something like this to have `ui` as a sub folder of `types`?

Reply:
Sounds good to me!